frontend/app/routes/$lang+/_public+/apply+/$id+/tax-filing.tsx

```diff
@@ -65,7 +71,7 @@ export async function action({ request, params }: ActionFunctionArgs) {
     state: { taxFiling2023: parsedDataResult.data },
   });
 
-  return parsedDataResult.data.taxFiling2023 === 'FALSE' ? redirectWithLocale(request, `/apply/${id}/file-your-taxes`, sessionResponseInit) : redirectWithLocale(request, `/apply/${id}/date-of-birth`, sessionResponseInit);
+  return parsedDataResult.data === 'FALSE' ? redirectWithLocale(request, `/apply/${id}/file-your-taxes`, sessionResponseInit) : redirectWithLocale(request, `/apply/${id}/date-of-birth`, sessionResponseInit);
```

Comment:
I suggest you early return instead of an inline IF.

```suggestion
  if (parsedDataResult.data === 'FALSE') { 
    redirectWithLocale(request, `/apply/${id}/file-your-taxes`, sessionResponseInit)
  }
   
  return redirectWithLocale(request, `/apply/${id}/date-of-birth`, sessionResponseInit);
```